pyhive/presto.py

```diff
@@ -307,14 +314,13 @@ def _fetch_more(self):
         """Fetch the next URI and update state"""
         self._process_response(self._requests_session.get(self._nextUri, **self._requests_kwargs))
 
-    def _decode_binary(self, rows):
-        # As of Presto 0.69, binary data is returned as the varbinary type in base64 format
-        # This function decodes base64 data in place
+    def _process_data(self, rows):
         for i, col in enumerate(self.description):
-            if col[1] == 'varbinary':
+            col_type = col[1].split("(")[0]
```

Comment:
Are column types always returned in lowercase? Might be safer to add a `.lower()` here.